pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>com.google.apis</groupId>
       <artifactId>google-api-services-androidpublisher</artifactId>
-      <version>v2-rev35-1.22.0</version>
+      <version>v2-rev45-1.22.0</version>
```

Comment:
Can you please update to v2-rev47-1.22.0? That's the latest working version for me [since 1.23.0 is throwing exception](https://issues.jenkins-ci.org/browse/JENKINS-31205?focusedCommentId=321406&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-321406).